### PR TITLE
Vospace 20210428 final

### PR DESCRIPTION
**Pull request by iglesu**
These changes fix a long standing bug of not deleting capabilities data when a user deletes the underlying identifier, improvements to the build process and changes to the vospace and IVOA capabilities base identifier (e.g. noao to noirlab)

### CODE REVIEW

File: java/vospace.properties.dldb1 [hunk 1]
```diff
@@ -11,13 +11,13 @@ backend.type.localfs=edu.caltech.vao.vospace.storage.LocalFSStorageManager
...
-server.http.url=http://dldb1.sdm.noao.edu:8080/vospace-2.0/vospace/data
+server.http.url=http://dldb1.datalab.noirlab.edu:8080/vospace-2.0/vospace/data
 
 # External address of transfers endpoint
-server.transfer.url=http://dldb1.sdm.noao.edu:8080/vospace-2.0/vospace/transfers
+server.transfer.url=http://dldb1.datalab.noirlab.edu:8080/vospace-2.0/vospace/transfers
 
 # External address of Datalab Authentication service
-server.auth.url=http://dlsvcs.datalab.noao.edu/auth/
+server.auth.url=http://dlsvcs.datalab.noirlab.edu/auth/
```
Comment (mjfitzpatrick) [inline java/vospace.properties.dldb1:20]: Preferred URL for production services uses 'https' and should be the proxy e.g. "https://datalab.noirlab.edu/auth" and not the specific 'dlsvcs' instance.

File: java/vospace.properties.gp04 [hunk 2]
```diff
@@ -11,13 +11,13 @@ backend.type.localfs=edu.caltech.vao.vospace.storage.LocalFSStorageManager
...
-server.http.url=http://dldb1.sdm.noao.edu:8080/vospace-2.0/vospace/data
+server.http.url=http://gp04.datalab.noirlab.edu:8080/vospace-2.0/vospace/data
 
 # External address of transfers endpoint
-server.transfer.url=http://dldb1.sdm.noao.edu:8080/vospace-2.0/vospace/transfers
+server.transfer.url=http://gp04.datalab.noirlab.edu:8080/vospace-2.0/vospace/transfers
 
 # External address of Datalab Authentication service
-server.auth.url=http://dlsvcs.datalab.noao.edu/auth/
+server.auth.url=http://dlsvcs.datalab.noirlab.edu/auth/
```
Comment (mjfitzpatrick) [inline java/vospace.properties.gp04:20]: https://datalab.noirlab.edu/auth

**Review by mjfitzpatrick — CHANGES_REQUESTED** [review]
Auth service URL should be proxy "https://datalab.noirlab.edu/auth" and not the 'http' specific instance.

File: java/vospace.properties.gp04 (hunk 2 above)
Comment (iglesu) [reply java/vospace.properties.gp04:20]: noted, will updated that. Thanks!

**Pull request merged**